pkg/config/parameters.go

```diff
@@ -333,8 +333,13 @@ func (p *Parameters) Validate() error {
 		return err
 	}
 
-	if err := p.Server.XDSServerVersion.Validate(); err != nil {
-		return err
+	if len(p.Server.XDSServerVersions) == 0 {
+		return fmt.Errorf("at least one xds server version must be specified")
+	}
+	for _, v := range p.Server.XDSServerVersions {
```

Comment:
it'd probably be good to de-dupe the list too, so `[v2, v2]` becomes just `[v2]`.

Reply:
I added a bit about de-duping in the validate method. Please take a look @skriss